### DISCUSSION TITLE
SPAM: SWiBoard

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [arendst/Tasmota#17815] [General]
Are you tired of having to physically be in the same room as your Tasmota devices to control them? Look no further! SWiBoard, the ultimate Tasmota control app, has got you covered. With its easy-to-use interface and seamless integration with MQTT, you can now control your devices from anywhere, at any time.

As the developer of SWiBoard, I would love to hear your feedback on the app. What are the features that you think are most important? What do you think could be improved? Your feedback will help me prioritize and add the features that matter most to you.

EDIT from Tasmota Crew: Link to PlayStore and advertising removed.
EDIT2: We recommend to use only tools where it can be verified the are safe and secure.

Thank you in advance for your valuable feedback!

![Screenshot_20230129-125405_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311621-081198ef-b244-43a8-846d-00bac36c25e6.png)

![Screenshot_20230129-125414_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311622-728d411a-7c38-4985-8d24-55142fbba6be.png)

![Screenshot_20230129-125411_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311619-e492b15b-aceb-4f28-af8b-5a73f5c563f1.png)

![Screenshot_20230129-125417_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311620-568ddf38-1e34-44ec-805c-5e40313cd8da.png)

![Screenshot_20230129-125408_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311618-808d12fa-ad96-40d9-8cc8-083a1c4fa341.png)

![Screenshot_20230129-125425_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311625-0945cc34-0c6d-4b56-91e8-6c295c68991c.png)

![Screenshot_20230129-125419_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311624-991bf99e-4eb4-4611-9daa-b459797973df.png)

![Screenshot_20230129-125422_Google Play Store.png](https://user-images.githubusercontent.com/12392345/215311623-e8cefee7-0338-4873-92d9-ca84c46ec4c6.png)

**Comment by TommySharpNZ** [comment]
Why did you spam all the other conversations???

**Reply by hemangjoshi37a** [reply]
Well I did very hard work developing it now it needs to be in attention for appropriate users. Is there anything problematic with it? And frankly that is not called spamming. That is called spreading awareness.

**Reply by TommySharpNZ** [reply]
No need to spam all the other conversations though. That has immediately turned me off your app and I will not be installing it.

**Reply by hemangjoshi37a** [reply]
Bro look at your GitHub profile. Because you are so negative nobody cares for you anyway. Have something positivity in your life.

**Reply by TommySharpNZ** [reply]
"Bro" you thanked us in advance for our feedback, yet my honest feedback seems to have you resorting to personal attacks.

I thank you for your feedback. I understand that you feel my GitHub profile does not reflect positivity, and I will make an effort to improve it. However, I would like to remind you that everyone is entitled to their own thoughts and feelings, and it is not productive to tell someone that nobody cares for them. Let's strive to be more supportive and understanding of each other.

**Reply by hemangjoshi37a** [reply]
Instead of bashing with me you could have typed few lines of code and should have made something positive. Instead here you are proving that a 4.9⭐ PlayStore app is a spam. I don't know brother get some help.

**Reply by s-hadinger** [reply]
Please stop this discussion. This is off-topic.

@hemangjoshi37a we have kindly asked you to stop spamming the other topics. Please stop pretending it is not spam, because it is. Again we **kindly** ask you to stop.

**Reply by hemangjoshi37a** [reply]
@s-hadinger if you are not interested in what I am proposing you can continue with your life. And regardless it is not spam at all I have exponential rise in my users base. I only thought it would be useful to tasmota community but it seems you guys don't deserve it. Very sad it turned out this bad.

**Comment by b1rigu** [comment]
Basically do I have to add the name of the AP that the tasmota device is broadcasting and it automatically connects to it? How did you do the connection? Because I tried to find a solution and it was all restricted on higher android aps and just plain manual.

**Reply by hemangjoshi37a** [reply]
No you have to connect to same network as tasmota device is connected. Then you have to enter the device topic of the tasmota device. And off you go. And there are lots of other functionalities like changing tasmota friendly device names then changing their icons etc. Feel free to ask if you have any problems using the app.

**Reply by hemangjoshi37a** [reply]
Here Tasmota device topic looks something like this `tasmota_A1B2C3` and that is located in information menu in tasmota web interface.

**Reply by hemangjoshi37a** [reply]
Also you have to add `broker.hivemq.com` in your tasmota MQTT settings to connect it to MQTT broker.
